corehq/apps/users/models.py

```diff
@@ -1142,20 +1142,11 @@ def from_old_couch_user(cls, old_couch_user, copy_id=True):
 
     @classmethod
     def wrap_correctly(cls, source):
-        if source['doc_type'] == 'CouchUser' and \
-                source.has_key('commcare_accounts') and \
-                source.has_key('web_accounts'):
-            from . import old_couch_user_models
-            # todo: remove this functionality and the old user models module
-            logging.error('still accessing old user models')
```

Comment:
Well that's not very useful! Running the search directly still returned no
results (at least none containing the search string)